src/components/timeline/duration.ts

```diff
@@ -0,0 +1,8 @@
+// From https://github.com/d3/d3-time/blob/main/src/duration.js
```

Comment:
To keep files more well-organized can we move this to the `utilities/timeline.ts` file? In general anything under `src/components` should just be `.svelte` files or test files.

Reply:
Yep! Can rearrange some things.